src/Nethermind/Nethermind.Config/BlocksConfig.cs

```diff
@@ -23,6 +23,8 @@ public class BlocksConfig : IBlocksConfig
 
         public ulong SecondsPerSlot { get; set; } = 12;
 
+        public ulong SimulateSecondsPerSlot { get; set; } = 1;
```

Comment:
Maybe it fits better in `JsonRcpConfig`?

Reply:
Yes, we can do it if we want to make it configurable.